### PR TITLE
MM-37809 Theming tweaks

### PR DESCRIPTION
**Pull request by matthewbirtch**
#### Summary
This PR makes adjustments to the default themes - particularly the dark themes as they have too much contrast and can be hard on the eyes.

#### Ticket Link
Jira https://mattermost.atlassian.net/browse/MM-37809

#### Screenshots
|  before  |  after  |
|----|----|
| image | image |
| image | image |
| image | image |
| image | image |
| image | image|

#### Release Note
```release-note
Updated the default themes to reduce eye strain (particularly on the dark themes)
```

### CODE REVIEW

**Comment by matthewbirtch** [comment]
Gentle ping to @asaadmahmood and @abhijit-singh a gentle ping to review this one

**Review by asaadmahmood — APPROVED** [review]
LGTM

**Review by devinbinnie — APPROVED** [review]

**Comment by lindalumitchell** [comment]
@matthewbirtch I compared theme codes in UI and visually verified between new and old. Checked the specific areas that changed, and clicked through the app in general. Looks great! A couple questions / verifications:

1. Should Playbooks UI also reflect the changes? Appears that in all themes, Playbooks Ui is using the old settings. Attaching a couple screenshots from Onyx.
2. I also noticed that a workspace preparation screen has some jarring element colors (but that might be a preexisting issue?). Attaching that screenshot as well.
3. Just to double-check: Tooltips should continue to appear dark black with light text, regardless of theme, as they do now, yes?

Note for my reference: I'm detailing in a comment on the Jira ticket the specific changes in each theme as listed in the Settings > Display UI.

playbooks PR - appears older colors
playbooks start PR - appears older colors
preparing-workspace

**Comment by matthewbirtch** [comment]
Thanks for reviewing in detail @lindalumitchell. Responses below

> * Should Playbooks UI also reflect the changes? Appears that in all themes, Playbooks Ui is using the old settings. Attaching a couple screenshots from Onyx.

I believe this requires a follow-up PR in the playbooks repo. In a conversation with @crspeller he mentioned "you might have to update the redux dependency in each one to change the default theme." Sounds like this will need to be a separate playbooks PR. Maybe Christopher can confirm.

> I also noticed that a workspace preparation screen has some jarring element colors (but that might be a preexisting issue?). Attaching that screenshot as well.

I don't think this has anything to do with the theme changes in this PR. Could you help create a ticket for this @lindalumitchell? I think this screen also appears out of order unexpectedly, but that's a different issue from the colors.

> Just to double-check: Tooltips should continue to appear dark black with light text, regardless of theme, as they do now, yes?

That's correct. No change to the tooltip colors here. They are always black with white text regardless of theme.

File: webapp/channels/src/packages/mattermost-redux/src/constants/preferences.ts [hunk 1]
```diff
@@ -92,9 +92,9 @@ const Preferences = {
             sidebarTextHoverBg: '#28427b',
             sidebarTextActiveBorder: '#5d89ea',
             sidebarTextActiveColor: '#ffffff',
-            sidebarHeaderBg: '#192a4d',
+            sidebarHeaderBg: '#192A4D',
```
Comment (hmhealey) [inline webapp/channels/src/packages/mattermost-redux/src/constants/preferences.ts:95]: Nitpicking, but it's weird to me that the existing RGB values are lower case and the new ones are upper case. Mind making those consistent and updating the snapshots?

**Review by hmhealey — APPROVED** [review]
One small request, but otherwise this looks good

File: webapp/channels/src/packages/mattermost-redux/src/constants/preferences.ts (hunk 1 above)
Comment (matthewbirtch) [reply webapp/channels/src/packages/mattermost-redux/src/constants/preferences.ts]: Updated in the latest commit

**Review by lindalumitchell — APPROVED** [review]
Thanks @matthewbirtch! Filed https://mattermost.atlassian.net/browse/MM-57957 for the workspace preparation screen.

If @crspeller agrees that the Playbooks updates should happen in another PR then I'm good with this one. Please let me know if you'd like me to create a new ticket for the Playbooks work as well.

**Comment by matthewbirtch** [comment]
Will need support from either @hmhealey or @devinbinnie to merge this

**Pull request merged**